Polyhedron/demo/Polyhedron/Plugins/PMP/Selection_plugin.cpp

```diff
@@ -843,7 +844,13 @@ public Q_SLOTS:
   {
     Scene_polyhedron_selection_item* selection_item = getSelectedItem<Scene_polyhedron_selection_item>();
     if(selection_item)
+    {
       selection_item->on_Ctrlz_pressed();
+      if(mode == 11)
```

Comment:
Is not it dangerous to hardcode a value `11`, instead of using a enumerator?

Reply:
Using 11 is dangerous because if the list of parameters of the widget is modified it will become the wrong number. Using an enum would do exactly the same. It would just make it clearer what 11 is supposed to be, but I commented everywhere what number is supposed to do.